plugins/wazuh-core/public/services/server-security/ui/components/format.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Wazuh app - React component for render user permissions requirements
+ * Copyright (C) 2015-2022 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React, { Fragment } from 'react';
+import { EuiSpacer } from '@elastic/eui';
+
+export const ServerPermissionsFormatted = permissions => {
+  return (
+    <div>
+      {permissions.map(permission => {
+        if (Array.isArray(permission)) {
+          return (
+            <div
+              key={`no-permissions-${getPermissionComponentKey(permission)}`}
+            >
+              <div>
+                - One of:{' '}
+                {permission
+                  .map(p =>
+                    PermissionFormatter(
+                      p,
+                      `no-permissions-${getPermissionComponentKey(
+                        permission,
+                      )}-${getPermissionComponentKey(p)}`,
+                    ),
+                  )
+                  .reduce((prev, cur) => [prev, ', ', cur])}
+              </div>
+              <EuiSpacer size='s' />
+            </div>
+          );
+        } else {
+          return (
+            <div
+              key={`no-permissions-${getPermissionComponentKey(permission)}`}
+            >
+              - {PermissionFormatter(permission)}
+            </div>
+          );
+        }
+      })}
+    </div>
+  );
+};
+
+const PermissionFormatter = (permission, key?: string) =>
+  typeof permission === 'object' ? (
+    <Fragment {...(key ? { key } : {})}>
+      <strong>{permission.action}</strong> (
+      <span style={{ textDecoration: 'underline' }}>{permission.resource}</span>
+      )
+    </Fragment>
+  ) : (
+    <strong {...(key ? { key } : {})}>{permission}</strong>
+  );
+
+const getPermissionComponentKey = permission =>
```

Comment:
Define before use it

Reply:
Fixed here: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7026/commits/1ada4ae8e6559e58c2d0de19f06489eed87cba71